src/metabase/api/common.clj

```diff
@@ -13,7 +13,7 @@
             [metabase.util :as u]
             [metabase.util.i18n :as i18n :refer [deferred-tru tru]]
             [metabase.util.schema :as su]
-            [schema.core :as s]
+            [schema.core :as schema]
```

Comment:
everywhere else, we see _both_ clojure.spec.alpha and schema.core as s: this vexes me a solidly moderate amount. perhaps we can cut the baby in half and just use both `schema` and `spec` and not have one with a single-letter?

Reply:
I actually did that at first but we have Kondo configured to have `s` be the canonical alias for `clojure.spec.alpha`, and nothing configured for `schema`. So I only did it this way to appease our Kondo configuration.

I agree about `spec/` and `schema/` respectively but changing that everywhere would be a big refactor and I don't want to do that right now